app/lib/controllers/chat_room_controller.dart

```diff
@@ -75,10 +113,10 @@ class ChatRoomController extends GetxController {
   Future<void> handleSendPressed(String message) async {
     // image or video is sent automatically
     // user will click "send" button explicitly for text only
-    await room.typingNotice(false);
-    var eventId = await room.sendPlainMessage(message);
+    await _room!.typingNotice(false);
```

Comment:
Don't think we need this now. As typing events are handled with stream.

Reply:
Yeah, it seemed to be strange for me too.
I wanted to keep original logic as much as possible.
If you don't need that code, I will remove it.

Reply:
Yes, please remove this.

Reply:
This code sends the typing notification back to server. But we don't need this now. The indicator will disappear automatically if the list is empty. Also this applies to chatlist.

Reply:
The list gets empty after some duration (~4 sec). The code is in typing indicator PR.